diff --git a/verdafile.mjs b/verdafile.mjs
index 4d4c99a..10a5800 100644
--- a/verdafile.mjs
+++ b/verdafile.mjs
@@ -92,7 +92,7 @@ const Ttc = phony(`ttc`, async t => {
 });
 
 const Ttf = phony(`ttf`, async t => {
-	await t.need(TtfFontFiles`TTF`, TtfFontFiles`TTF-Unhinted`);
+	await t.need(TtfFontFiles`TTF-Unhinted`);
 });
 
 const CheckTtfAutoHintExists = oracle("oracle:check-ttfautohint-exists", async target => {
